README.md

```diff
@@ -1,35 +1,89 @@
 # podannotator
-Annotate pods with PVC in a given list of namepsaces to be backed up by velero/restic
+Annotates all volumes (with a PVC), in all pods, in a given list of namepsaces to be backed up by velero/restic.
 
+Namespace 1
```

Comment:
Please take a look on how the output looks like. https://github.com/markhankins/podannotator/tree/v2
You may want to ad 3 quotes simbols to preserve the spacing:

```
        -- Pod 2
                -- Volume 1 (N1P2V1)
                -- Volume 2 (N1P2V2)
```

Reply:
Have changed it to indented lists.